ingest/jobs/Get_Ingest_Config/tests/end_to_end/features/fixtures.py

```diff
@@ -0,0 +1,30 @@
+from azure.identity import DefaultAzureCredential
+from azure.storage.filedatalake import DataLakeServiceClient
+from behave import fixture
+from constants import ADLS_URL, RAW_CONTAINER_NAME, AUTOMATED_TEST_OUTPUT_DIRECTORY_PREFIX
+from utils.azure.adls import filter_directory_paths_adls, delete_directories_adls
+
+@fixture
+def azure_adls_clean_up(context, ingest_directory_name: str):
+    credential = DefaultAzureCredential()
+    adls_client = DataLakeServiceClient(account_url=ADLS_URL, credential=credential)
+    print('BEFORE SCENARIO. DELETING ANY AUTOMATED TEST OUTPUT DATA')
+    automated_test_output_directory_paths = filter_directory_paths_adls(adls_client,
+                                                                        RAW_CONTAINER_NAME,
+                                                                        ingest_directory_name,
+                                                                        AUTOMATED_TEST_OUTPUT_DIRECTORY_PREFIX)
+
+    if automated_test_output_directory_paths:
```

Comment:
this `if` is redundant - can you please remove here and in other places?

Reply:
@lorraine-houston-ensono Could you please check this

Reply:
Again, this is in our preexisting tests. I'll take a look tomorrow but this shouldn't be a blocker for this PR

Reply:
`filter_directory_paths_adls` returns a list and `delete_directories_adls` deletes paths from the list. If empty -- nothing will be deleted. There are six occurrences of this `if`, and all look redundant to me. I think removing redundant code saves more time than it costs :)

Reply:
Done

Reply:
great, thanks!